tools/proteore_venn_diagram/venn_diagram.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python2.7
```

Comment:
python 2 is depreated ...

Can we replace this tool wth https://github.com/peterjc/pico_galaxy/blob/master/tools/venn_list/venn_list.xml?